packages/react-devtools-core/src/backend.js

```diff
@@ -17,13 +17,6 @@ type ConnectOptions = {
   isAppActive?: () => boolean,
 };
 
-// TODO: why?
-if (!window.performance) {
```

Comment:
The only place depending on `performance.now` seems to be Relay plugin so inlined this.